kona/run-kona-host.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# This is a shim for kona-host used as the --cannon-kona-server value when running op-challenger in acceptance tests
```

Comment:
We could be more selective in enabling when `presets.WithCannonKona()` is used. Only tests that actually  run kona in the VM should have that specified.

If the op-challenger only configured the cannon-kona trace type when an acceptance test enables `presets.WithCannonKona()`, would that eliminate the need for this shim?

Reply:
Managed to only set the cannon kona config when the game is actually being added so we don't need the run-kona-host.sh script now.